WISE-Community/WISE
# Build the angular site in production mode

How to test:
1. Run ./wise.sh package. This will build the angular site in production mode before building target/wise.war.
2. unzip wise.war. 
3. Verify that mainXYZ.js is inside site/dist folder. XYZ should be a randomly-generated string.
4. Open mainXYZ.js and verify that its contents are minified.

Closes #1558
